client/src/pages/SignUp.js

```diff
@@ -12,17 +12,48 @@ import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
 import Typography from '@mui/material/Typography';
 import Container from '@mui/material/Container';
 import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { useMutation } from "@apollo/react-hooks";
+
 
 const theme = createTheme();
 
+const SIGNUP_MUTATION = gql`
```

Comment:
We shouldn't have the individual mutations and queries in the react pages. We should be storing them in utilities file and exporting. We need to keep this in client > src > util > mutations.js